deployments/qat_dpdk_app/crypto-perf/crypto-perf-dpdk-pod-requesting-qat-cy.yaml

```diff
@@ -1,14 +1,18 @@
 kind: Pod
 apiVersion: v1
 metadata:
-  name: qat-dpdk
+  name: qat-dpdk-test-crypto-perf
 spec:
   containers:
   - name: crypto-perf
-    image: intel/crypto-perf:devel
+    image: intel/dpdk-test-perf:devel
     imagePullPolicy: IfNotPresent
-    command: [ "/bin/bash", "-c", "--" ]
-    args: [ "while true; do sleep 300000; done;" ]
+    command: [ "/bin/bash", "-c", "/usr/bin/run-dpdk-test" ]
```

Comment:
`command` is redundant since you can use the image `ENTRYPOINT` as is.

Reply:
Could you check again? I do not understand how command can be redundant when there is no `base` anymore..!

Reply:
https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#define-a-command-and-arguments-when-you-create-a-pod

in other words: `command` is not needed.